templates/bundles/PrestaShopBundle/Admin/Configure/AdvancedParameters/Email/Blocks/email_configuration.html.twig

```diff
@@ -42,8 +42,8 @@
           {{ form_row(emailConfigurationForm.log_emails) }}
           <div class="alert alert-info" role="alert">
             <p class="alert-text">
-              {{ 'Fill in the fields below to set up DKIM signing of outgoing emails. This will reduce the likelihood of your emails being marked as spam. 
```

Comment:
Isn't this gonna change the wording key?

Reply:
Possibly, yeah... But I guess it's better without the space and we can add the new key, right?

Reply:
Mmm seems like it's a new wording from develop branch so we can change it However the weirdest part here is to have line breaks inside the wording. The wording should be on a single line.